### DISCUSSION TITLE
Add certbot-dns-dynu (Dynu DNS) plugin for Let's Encrypt DNS challenge as an option.

### DISCUSSION THREAD

**Discussion by dexterdar** [hassio-addons/addon-nginx-proxy-manager#317] [Ideas]
I love using Dynu DNS with Let's Encrypt as it allows me to use wildcards for the SSL cert (*.mydomain.com) and they also provide an API key for using DNS challenges.

I currently use pfsense to manage my certs and renewal + haproxy for SSL reverse proxy.  It would be great to include Dynu DNS as an option for the DNS challenge for Let's Encrypt SSL cert management so that I could migrate my pfsense functionality to Nginx proxy manager and remove the pfsense device entirely from the equation.

I found that there is a certbot plugin made for Dynu DNS already and was wondering if it would be possible to integrate it into this project?  

https://github.com/bikram990/certbot-dns-dynu

**Comment by frenck** [comment]
![CleanShot 2024-01-13 at 11 12 10@2x](https://github.com/hassio-addons/addon-nginx-proxy-manager/assets/195327/66711b44-7c62-4eb4-b644-9bf3cf84fd71)

Will be part of the upcoming/next release of this add-on.

../Frenck
